statusHandler/statusMetricsProvider.go

```diff
@@ -95,12 +96,44 @@ func (nd *statusMetrics) Close() {
 }
 
 // StatusMetricsMap will return all metrics in a map
-func (nd *statusMetrics) StatusMetricsMap() (map[string]interface{}, error) {
+func (nd *statusMetrics) StatusMetricsMap() map[string]interface{} {
 	statusMetricsMap := make(map[string]interface{})
 	nd.nodeMetrics.Range(func(key, value interface{}) bool {
 		statusMetricsMap[key.(string)] = value
 		return true
 	})
 
-	return statusMetricsMap, nil
+	return statusMetricsMap
```

Comment:
Fix tests on `elrond-go/statusHandler/statusMetricsProvider_test.go` . the teamcity build for tests failed

Reply:
:+1: done